src/index.ts

```diff
@@ -28,6 +28,6 @@ export {
 } from "./simularium";
 export { compareTimes, loadSimulariumFile } from "./util";
 export { DEFAULT_CAMERA_SPEC } from "./constants";
-export { AgentData } from "./simularium/types";
+export type { AgentData } from "./simularium/types";
```

Comment:
small bug fix folded in here

Reply:
this should be included in the types imports up above and should be exported directly from "./simularium" like the rest of them are. probably should be it's own PR because currently it breaks running the viewer on main

Reply:
thank you, [PR here.](https://github.com/simularium/simularium-viewer/pull/401)